Otherwise it eats the CPU and performance.
The Vosk devs set the same but with flags at compile timeflags.
---
diff --git a/src/model.cc b/src/model.cc
index 035ffee..3ee40b0 100644
--- a/src/model.cc
+++ b/src/model.cc
@@ -29,6 +29,7 @@
 // We need to set num threads
 #include <mkl.h>
 #endif
+#include <cblas.h>
 
 namespace fst {
 
@@ -117,6 +118,8 @@ Model::Model(const char *model_path) : model_path_str_(model_path) {
     mkl_set_num_threads(1);
 #endif
 
+    openblas_set_num_threads(1);
+
     struct stat buffer;
     string am_v2_path = model_path_str_ + "/am/final.mdl";
     string model_conf_v2_path = model_path_str_ + "/conf/model.conf";
